src/diff.ts

```diff
@@ -0,0 +1,28 @@
+import crypto from 'crypto';
+import * as core from '@actions/core';
+import { Repo } from './github';
+import { VersionResponse } from 'bump-cli';
+import { bumpDiffComment } from './common';
+
+const commentTitle = '🤖 Documentation diff detected:';
+const codeBlock = '```';
+const poweredByBump = '> _Powered by [Bump](https://bump.sh)_';
```

Comment:
```suggestion
const poweredByBump = '_Automatic API diff by [Bump](https://bump.sh)_';
```

For SEO purpose (I removed the quote, I think that italic is enough, but if you prefer just re-add it!) ;)

Reply:
I don't think it affects SEO to have `>` in a github comment, does it really?

I think it's cleaner in a quote rather than simply in italic. To really differentiate the information which is less important than the actual diff.

Reply:
More than SEO, it adds more context about what we really do, so I think that the wording change does make sense here.

OK for the quote. I'm not a big fan of it, but if you think it's better, let's keep it!

Reply:
Ah ok for the wording, however it repeats a bit too much “API” word doesn't it? Here's what it would look like:

---

🤖 API change detected:
```
Added: GET /versions/{version_id}
```

[View visual diff](http://dev.bump.sh:3000/changes/6c1b8f18-ad8a-4902-9b61-691bb375d121)

> _Automatic API diff by [Bump](https://bump.sh)_

---

I think I prefer this version:

---

🤖 API change detected:
```
Added: GET /versions/{version_id}
```

[View visual diff](http://dev.bump.sh:3000/changes/6c1b8f18-ad8a-4902-9b61-691bb375d121)

> _Powered by [Bump](https://bump.sh)_

---

What do you think @antlio?

Reply:
Let's keep it.

Reply:
sorry i'm late on this! let's stick to that from now on :) 👏